viewer/apiSessions.js

```diff
@@ -1838,7 +1838,7 @@ class SessionAPIs {
 
       Promise.all([
         Db.searchSessions(indices, query, options),
-        Db.numberOfDocuments(['sessions2-*', 'sessions3-*'], options.cluster ? { cluster: options.cluster } : {})
+        Db.numberOfDocuments(Db.defaultIndexPatterns(Config.getArray('queryExtraIndices', '')), options.cluster ? { cluster: options.cluster } : {})
```

Comment:
Throughout the code the several places where `['sessions2-*', 'sessions3-*']` are passed into db.js have been replaced by `Db.defaultIndexPatterns` which takes an `extraIndices` array (pulled from the config's `queryExtraIndices` value) and merges its contents with the default `['sessions2-*', 'sessions3-*']` array. This merged array is returned.